src/p2a_analyses_read_processed_data.R

```diff
@@ -107,27 +107,32 @@ data <- data[-which(data$name == "Ben" & data$dt == as.POSIXct(strptime("2016-11
 #### STEP 4: ANNOTATE GEOGRAPHICAL INFORMATION
 ################################################
 library(rgdal);library(sp);library(raster)
+
+if(!dir.exists("data/maps/ne_50m_admin_0_countries")) {
```

Comment:
I would keep consistency in the usage of the `here` package and also use it in these sections similar to the other adaptations

Reply:
That would make more sense indeed. The code has been changed using the here function.